shm_wrapper/print_shm.c

```diff
@@ -0,0 +1,16 @@
+#include "shm_wrapper.h"
+
+int main(int argc, char* argv[]) {
+    uint32_t devices = 0;
+    if (argc == 1) {
+        devices = -1;
+    }
+    else {
+        for (int i = 1; i < argc; i++) {
+            devices |= (1 << atoi(argv[i]));
+        }
+    }
+    logger_init(TEST);
+    shm_init(TEST);
+    print_params(devices);
```

Comment:
should also `shm_stop(TEST)` and `logger_stop(TEST)` here just to be explicit... especially if logging to network (not sure how hanging open pipes will behave--probably it's ok because a process closes all file descriptors when it exits, but just to be safe)

Reply:
Yea forgot about them. I'll add them